src/coreclr/pal/src/thread/context.cpp

```diff
@@ -227,7 +268,11 @@ BOOL CONTEXT_GetRegisters(DWORD processId, LPCONTEXT lpContext)
 
     if (processId == GetCurrentProcessId())
     {
+#ifndef HOST_LOONGARCH64
         CONTEXT_CaptureContext(lpContext);
+#else
+        _ASSERT(!"unimplemented on LOONGARCH64 yet");
```

Comment:
Should this be implemented in this PR?

Reply:
OK.
I had delete the assert for having implements of the `CONTEXT_CaptureContext`.